cmd/daemon/main.go

```diff
@@ -17,9 +21,24 @@ import (
 
 const runtimeTypeContainerd string = "containerd"
 
+type config struct {
```

Comment:
I think's it better to define it in pkg

Reply:
Why is that? Given that these values aren't used outside of main, I'd be inclined to leave them here, similar to how [serving does it](https://github.com/knative/serving/blob/ef89ff8061a935c44a283c07f872d47b57204089/cmd/queue/main.go#L67-L103).

Reply:
make sense